internal/pkg/agent/install/uninstall.go

```diff
@@ -97,8 +97,8 @@ func Uninstall(cfgFile, topPath, uninstallToken string) error {
 // to an ERROR_SHARING_VIOLATION. RemovePath will retry up to 2
 // seconds if it keeps getting that error.
 func RemovePath(path string) error {
-	const arbitraryTimeout = 2 * time.Second
-	var start time.Time
+	const arbitraryTimeout = 5 * time.Second
+	start := time.Now()
```

Comment:
Not sure if it's relevant but before the change we would start counting time from the first error, now we do it even before the first attempt...

One possible side effect is that if the first `os.RemoveAll` takes longer than than the timeout we would not retry this.